static/js/theme-map/Maps/Providers/Baidu.js

```diff
@@ -350,7 +350,7 @@ function load(resolve, reject, apiKey, {
   }
 
   const negativeLngFixStyle = document.createElement('style');
-  negativeLngFixStyle.innerHTML = negativeLngFixCSS;
+  negativeLngFixStyle.innerText = negativeLngFixCSS;
```

Comment:
we may want `textContent` here just like we did for the `InlineAssetHtmlPlugin` because it's also adding css to a style tag. to test this, you might need to set up a Baidu map in our test-site. I don't believe we currently have one

Reply:
tested a Baidu map in the test site and saw that pins with negative longitude show up as expected even after zooming in and out